tests/default.nix

```diff
@@ -95,6 +95,7 @@ import nmt {
     ./modules/programs/newsboat
     ./modules/programs/nix-index
     ./modules/programs/nnn
+    ./modules/programs/nheko
```

Comment:
"h" comes before "i" so this should be between newsboat and nix-index

Reply:
damn, a hiccup 🥲